doc/man3/SSL_CTX_set_alpn_select_cb.pod

```diff
@@ -119,14 +154,21 @@ ALPN protocol not selected.
 
 =back
 
+The SSL_CTX_set_next_proto_select_cb() callback should return
+B<SSL_TLSEXT_ERR_OK> if successful. Any other value is fatal to the connection.
```

Comment:
Do you mean the function set with set with SSL_CTX_set_next_proto_select_cb(), so `cb`? Since this is not the return value of SSL_CTX_set_next_proto_select_cb() itself, I think it should get documented where you describe the callback.